src/components/Avatar/Avatar.component.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Avatar() {
+  return (
+    <>
+      <AvatarPlaceholder />
+    </>
+  );
+}
+
+export default Avatar;
+
+const AvatarPlaceholder = styled.div`
+  width: 2.5rem;
+  height: 2.5rem;
+  border-radius: 50%;
+  background-color: #e5e7eb;
+`;
```

Comment:
The react fragment is only necessary when you will render more than one node, on this case, is just the `AvatarPlaceholder `. 

I believe that on the future you will add more elements and that's why there is the react fragment